uktrade/data-hub-frontend
# Feature/ccit 128 ess interaction on contact page

## Description of change

ESS Interactions show on the Company activity Feed already. This PR also shows ESS Interactions on the CONTACT activity feed. 

## Test instructions

Go to a company with ESS interactions, e.g. One List Corp, select a contact from the ESS Interaction (e.g. Test Test), view their Interactions on their Contact page. The same ESS interaction will appear in the CONTACT activity feed. 
e.g. http://localhost:3000/contacts/c1376844-89da-409c-8194-13e0b509b9d0/interactions


## Screenshots

### Before
![Screenshot 2023-01-30 at 15 17 54](https://user-images.githubusercontent.com/72502389/215517531-16a89857-66dc-4d57-9fa7-51386f98469d.png)


### After
![Screenshot 2023-01-30 at 15 16 41](https://user-images.githubusercontent.com/72502389/215517599-3c4542ce-deeb-437d-bdca-ecc2365c9ea9.png)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
